src/components/FiatValue.svelte

```diff
@@ -0,0 +1,47 @@
+<script>
+  import { CoinGeckoId, FiatSymbol } from '../constants';
+
+  export let coin;
+  export let amount;
+  export let fiat;
+
+  async function getValue(id, amount, currency) {
+    if (!id || !amount || !currency) return;
+
+    const endpoint = 'https://api.coingecko.com/api/v3/coins/markets';
+    const res = await fetch(`${endpoint}?vs_currency=${currency}&ids=${id}`);
+    const json = await res.json();
+
+    if (res.ok) {
+      const value = (json[0].current_price * amount).toFixed(2);
+      return `${FiatSymbol[fiat]} ${value}`;
+    } else {
+      throw new Error(json);
+    }
```

Comment:
```suggestion
    if (!res.ok) throw new Error(json);
    
    const value = (json[0].current_price * amount).toFixed(2);
    return `${FiatSymbol[fiat]} ${value}`;
```

Reply:
nit: just cosmetic preference, if we can avoid else branching why not

https://www.yegor256.com/2016/08/10/if-then-else-code-smell.html